services/src/main/scala/org/apache/gearpump/services/AppMasterService.scala

```diff
@@ -34,7 +34,7 @@ import scala.util.{Failure, Success, Try}
 
 trait AppMasterService extends HttpService  {
   import upickle._
-  val master:ActorRef
+  def master:ActorRef
```

Comment:
@manuzhang 

There is a contention.

Worker still have not register itself to master(because it start in async way) when the error happen.